tools/dbTools.py

```diff
@@ -115,9 +78,16 @@ def is_scraped(self, file_hash):
         )
         result = self.cursor.fetchone()
         if result:
-            return result.id, result.scrape_again
+            return {
+                 'id': result.id,
+                 're-scrape': result.scrape_again,
+                 'status': True
```

Comment:
why do you introduce a new field `status`?

Reply:
Because I want to be able to check if we need to scrape the item because it's not in the database (`status: False`) or because it is in the database, but `scrape_again` is true (`status: True, scrape_again: True`) in which case I need to use the id in the dict.

Reply:
And `result.id` does not contain that information? I would expect this to be null when it does not exist in the database.